eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteChannelsDb.scala

```diff
@@ -170,6 +177,27 @@ class SqliteChannelsDb(val sqlite: Connection) extends ChannelsDb with Logging {
     }
   }
 
+
+  override def listClosedChannels(remoteNodeId_opt: Option[PublicKey], paginated_opt: Option[Paginated]): Seq[PersistentChannelData] = withMetrics("channels/list-closed-channels", DbBackends.Sqlite) {
+    val sql = "SELECT data FROM local_channels WHERE is_closed=1 ORDER BY closed_timestamp"
+    remoteNodeId_opt match {
```

Comment:
For postgres you don't have to do the `remote_node_id` filtering manually, there is a `remote_node_id` and an index on that field. You should use a different SQL statement when filtering by node_id, something like:

```
val sql = remoteNodeId_opt match {
  case None => "SELECT data FROM local_channels WHERE is_closed=1 ORDER BY closed_timestamp DESC"
  case Some(remoteNodeId) => s"SELECT data FROM local_channels WHERE is_closed=1 AND remote_node_id = ${remoteNodeId.toHex} ORDER BY closed_timestamp DESC"
}
```

And then add the `limited` if pagination is defined.

Note: like for sqlite, shouldn't we filter by descending `closed_timestamp`?